### ISSUE TITLE
Question about the mask token id and sos token id

### ISSUE THREAD

**Issue by larryzhang23** [dome272/MaskGIT-pytorch#9]
Hi,
In transformer.py, I find mask_token_id is set to be args.num_image_tokens. Shouldn't it be the args.num_codebook_vectors? I think we don't want the mask token id to be one of those in the codebook. Similar thing for the sos token id.

**Comment by dome272** [comment]
Oh yea thats correct. My bad. Thanks for correcting. Although it doesnt seem to harm performance that bad, Im currently training another model at 32x32 resolution. Thats the loss curve:
![image](https://user-images.githubusercontent.com/61938694/164944343-8adb73e0-c8eb-4e5d-9a8e-86f2bba6e072.png)

**Comment by larryzhang23** [comment]
The loss curve seems not bad

**Comment by dome272** [comment]
I restarted training and it seems to perform better now. 
![image](https://user-images.githubusercontent.com/61938694/165140771-4d7a07a8-7dc7-4d03-96e9-4c0a4f518f60.png)
(Green is the fixed tokens)
But I also changed something at the optimiter weight decay, so maybe this also has some effect on it.
